examples/machinedeployment/machinedeployment.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     cluster.x-k8s.io/cluster-name: ${CLUSTER_NAME}
     nodepool: nodepool-0
 spec:
-  replicas: 1
+  replicas: 2
```

Comment:
is this a cluster only with 2 nodes?

Reply:
2 worker nodes (minimum required for conformance testing), there should also be 3 control plane machines as well.

Reply:
1 bastion node, 3 controllers, 2 workers (with this change included)

Reply:
is there a load balancer in front of the controllers for the APIs?